packages/loader/test-loader-utils/package.json

```diff
@@ -15,9 +15,14 @@
 	"module": "dist/index.js",
 	"types": "dist/index.d.ts",
 	"scripts": {
+		"api": "fluid-build . --task api",
+		"api-extractor:commonjs": "api-extractor run --local",
+		"api-extractor:esnext": "copyfiles -u 1 \"dist/**/*-@(alpha|beta|public|untrimmed).d.ts\" lib",
 		"build": "fluid-build . --task build",
 		"build:compile": "fluid-build . --task compile",
-		"clean": "rimraf --glob dist \"**/*.tsbuildinfo\" \"**/*.build.log\"",
+		"build:docs": "fluid-build . --task api",
```

Comment:
Do we need to start building docs for this?

Reply:
I would argue that it's useful to use API-Extractor in all of our library packages, regardless of if they're private or not. That said, we probably don't need to enforce the presence of tags or enable any trimming for `fluid-private` packages.